packages/server/tests/acceptance/rpc.spec.ts

```diff
@@ -406,8 +407,10 @@ describe('RPC Server Acceptance Tests', function () {
                     const transaction = {
                         ...default155TransactionData,
                         to: mirrorContract.evm_address,
-                        nonce: await relay.getAccountNonce(accounts[2].address)
+                        nonce: await relay.getAccountNonce(accounts[2].address),
```

Comment:
Why was lack of gasPrice working before.
Please add to comments section of PR something to highlight this issue and solution